api/models/Build.js

```diff
@@ -1,12 +1,58 @@
-/**
-* Build.js
-*
-* @description :: TODO: You might write a short summary of how this model works and what it represents here.
-* @docs        :: http://sailsjs.org/#!documentation/models
-*/
+var afterCreate = (model, done) => {
```

Comment:
We should probably consider removing the messaging logic from this lifecycle callback and instead rely on a build inside of a controller action where builds are created.